sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationLiveTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class TranslationOperationLiveTests : DocumentTranslationLiveTestBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TranslationOperationLiveTests"/> class.
+        /// </summary>
+        /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
+        public TranslationOperationLiveTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        private List<string> Documents = new List<string>
+        {
+            "This is the first english test document",
+            "This is the second english test document"
+        };
+
+        [RecordedTest]
+        public async Task TranslationOperationTestAsync()
```

Comment:
nit: the name of the tests doesn't need to include Async as our Test Framework runs the tests in Async and sync mode. This is why you see 2 recording files one with this name and one with `nameAsync` so looks like `TranslationOperationTestAsyncAsync`.